rasa/core/test.py

```diff
@@ -604,7 +604,7 @@ def plot_story_evaluation(
     from sklearn.metrics import confusion_matrix
     from sklearn.utils.multiclass import unique_labels
     import matplotlib.pyplot as plt
-    from rasa.nlu.test import plot_confusion_matrix
```

Comment:
A bit unlikely that it happens but it could break someone's code relying on this function in some glue code of their own. Should we add a deprecation warning or keep a dummy function in `rasa.nlu.test` which just calls the one in `rasa.utils.plotting`?

Reply:
Not sure if that is needed as it will be released with 2.0. I also removed two cli options for naming the histogram and confusion matrix file which are not useful anymore. If you think it is needed, I'll leave those and add a deprecation warning.

Reply:
@tmbo Just wanted to double check if this is fine.